src/System.Net.Security/ref/4.0.0/System.Net.Security.cs

```diff
@@ -0,0 +1,162 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Yes that is fairly recently @ericstj did some tooling so we don't need to keep these older builds actually compiling in the repo and instead just grab them from our last stable package.